src/main/java/net/rptools/maptool/client/ui/zone/ZoneRenderer.java

```diff
@@ -1186,7 +1213,7 @@ public void renderZone(Graphics2D g2d, PlayerView view) {
       List<DrawnElement> drawables = zone.getBackgroundDrawnElements();
       // if (!drawables.isEmpty()) {
       timer.start("drawableBackground");
-      renderDrawableOverlay(g2d, backgroundDrawableRenderer, view, drawables);
+      if (!skipDrawing) renderDrawableOverlay(g2d, backgroundDrawableRenderer, view, drawables);
```

Comment:
If statements in this source file don't follow style guide

Reply:
Changed.